ui/lib/apps/SlowQuery/components/SlowQueriesTable.tsx

```diff
@@ -62,7 +62,7 @@ export default function SlowQueriesTable({
       columns={columns}
       items={slowQueries}
       onRowClicked={handleRowClick}
-      getKey={(row) => row?.digest}
+      getKey={(row) => row && `${row.digest}_${row.timestamp}`}
```

Comment:
`(row) => `${row?.digest}_${row?.timestamp}` will warning duplicated keys when loading because all keys are `undefined_undefined`, so change to `row && `${row.digest}_${row.timestamp}`